src/ai/susi/mind/SusiSkill.java

```diff
@@ -525,12 +525,42 @@ public static JSONObject getSkillMetadata(String model, String group, String lan
                     JSONObject languageName = groupName.getJSONObject(language);
                     if (languageName.has(skillname)) {
                         JSONObject skillName = languageName.getJSONObject(skillname);
+                        if (!skillName.has("stars")){
+                            JSONObject newFiveStarRating=new JSONObject();
+                            newFiveStarRating.put("one_star", "0");
+                            newFiveStarRating.put("two_star", "0");
+                            newFiveStarRating.put("three_star", "0");
+                            newFiveStarRating.put("four_star", "0");
+                            newFiveStarRating.put("five_star", "0");
+                            newFiveStarRating.put("avg_star", "0");
+                            newFiveStarRating.put("total_star", "0");
+
+                            skillName.put("stars", newFiveStarRating);
+                        }
                         skillMetadata.put("skill_rating", skillName);
                     }
+                    else {
+                        JSONObject newRating=new JSONObject();
+                        newRating.put("negative", "0");
+                        newRating.put("positive", "0");
+
+                        JSONObject newFiveStarRating=new JSONObject();
+                        newFiveStarRating.put("one_star", "0");
+                        newFiveStarRating.put("two_star", "0");
+                        newFiveStarRating.put("three_star", "0");
+                        newFiveStarRating.put("four_star", "0");
+                        newFiveStarRating.put("five_star", "0");
+                        newFiveStarRating.put("avg_star", "0");
+                        newFiveStarRating.put("total_star", "0");
+
+                        newRating.put("stars", newFiveStarRating);
```

Comment:
Same feedback here. Please consider using integers as the value for each key in the object. These are mathematical and would be nice to have that type of computation being done here.

Reply:
I've followed the convention that is used in the thumbs feedback system. It stores the negative and position feedback count as string. So I think we should go ahead with this convention only. Also parseInt() is used at every point where the value has been accessed.